runner/operator/access/v1_0_0/snps_and_indels/test/test_snps_and_indels_operator.py

```diff
@@ -0,0 +1,99 @@
+import os
+
+from django.test import TestCase
+from django.core.management import call_command
+
+from beagle.settings import ROOT_DIR
+from beagle_etl.models import Operator
+from file_system.models import File, FileMetadata
+from runner.operator.operator_factory import OperatorFactory
+from runner.operator.access.v1_0_0.snps_and_indels import AccessLegacySNVOperator
+
+
+CURDIR = os.path.dirname(os.path.realpath(__file__))
+TEST_FIXTURE_DIR = os.path.join(CURDIR, 'fixtures')
+FIXTURES = [
+    "curated_normal_files.json",
+    "curated_normals_file_group.json",
+    "curated_normals_file_metadata.json",
+    "files.json",
+    "files_metadata.json",
+    "operator_run.json",
+    "ports.json",
+    "runs.json",
+]
+
+# Todo: how to shorten these to remove folder names like other tests have it?
```

Comment:
I think its because you have them in the wrong place in the directory tree

Fixtures that are global for Beagle such as File Groups and File Types can be moved up to the other fixtures for those in the root of the app dir (`file_system/fixtures`) and added to the JSON's there. When the fixtures are in that location it should be able to load automatically without the full file path like the other items in the `fixtures` attribute on the test case class.

Fixtures that are sample-specific such as Runs, Ports, Files, FileMetadata, could be included in the project root fixtures dir (`beagle/fixtures`) but I dont think there was much organization or consistent labeling of those so you would be free to set up the dir for your ACCESS fixtures however you like. Note that you would still have to resolve the full path to them when you load them in the test case though, I think.

Reply:
I've moved these fixtures to the `beagle/fixtures` dir, and removed the FileGroup fixture file in favor of adding a FileGroup to the common fixtures file. So I believe this should be resolved now!